charts/cp-schema-registry/values.yaml

```diff
@@ -69,6 +97,12 @@ tolerations: []
 ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
 
+securityContext:
+  runAsUser: 10001
```

Comment:
Don't forget to put a comment saying why this number

Reply:
I don't really know why, I could put it's a security team recommendation but Jérémy couldn't tell me the reason why
I added some infos in the readme of the chart, following the "convention" used by the confluent chart, they add details about default values in the readme

Reply:
I've added some details